pyrobosim/examples/demo.py

```diff
@@ -122,14 +122,16 @@ def create_world(multirobot=False):
             grasp_generator=GraspGenerator(grasp_props),
         )
         world.add_robot(robby, loc="bedroom")
-        robby.set_path_planner(
-            RRTPlanner(world, bidirectional=True, rrt_connect=False, rrt_star=True)
-        )
+        planner_config = {
+            "grid": None,
+            "world": world,
+            "bidirectional": False,
+            "rrt_connect": False,
+            "rrt_star": False,
+        }
+        rrt = PathPlanner("rrt", planner_config)
+        robby.set_path_planner(rrt)
```

Comment:
This is only adding a planner in the multirobot case. When I ran the demo without any extra arguments, I got a warning that the first robot (simply named "robot"), doesn't have a planner.

I guess a byproduct of this work is that if you instantiate a robot by default, it doesn't have a planner. So basically none of the demos actually work in their current state. Please check this.

When I did run this file with the `--world-file test_world.yaml` argument, it works but for some reason navigating to `counter0` kept failing. Also check this.